Dockerfile.android

```diff
@@ -1,13 +1,35 @@
-FROM godot-fedora:latest
+ARG mono_version
+FROM godot-mono:${mono_version}
+ARG mono_version
 
 RUN dnf -y install scons java-1.8.0-openjdk-devel ncurses-compat-libs unzip which gcc gcc-c++ && \
+    mkdir sdk && cd sdk && \
     curl -LO https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip && \
     unzip sdk-tools-linux-4333796.zip && \
     rm sdk-tools-linux-4333796.zip && \
     yes | tools/bin/sdkmanager --licenses && \
-    tools/bin/sdkmanager ndk-bundle 'platforms;android-23' 'build-tools;19.1.0' 'build-tools;28.0.3' 'platforms;android-28' 
+    tools/bin/sdkmanager ndk-bundle 'build-tools;28.0.3' 'platforms;android-28' 'cmake;3.10.2.4988404' && \
+    cd ..
 
-ENV ANDROID_HOME=/root/
-ENV ANDROID_NDK_ROOT=/root/ndk-bundle/
+ENV ANDROID_HOME=/root/sdk/
+ENV ANDROID_NDK_ROOT=/root/sdk/ndk-bundle/
+
+RUN dnf -y install git patch && \
+    if [ ${mono_version%%.*} -ge 6 ]; then \
+        curl https://download.mono-project.com/sources/mono/mono-${mono_version}.tar.xz | tar xJ; \
+    else \
+        curl https://download.mono-project.com/sources/mono/mono-${mono_version}.tar.bz2 | tar xj; \
+    fi && \
+    cd mono-${mono_version} && \
+    mono_sources=$(pwd) && \
+    curl -LO https://raw.githubusercontent.com/godotengine/godot/master/modules/mono/build_scripts/patches/fix-mono-android-tkill.diff && \
```

Comment:
Can you place this patch in files/ instead?

Reply:
Done.